docs/testplan/Smartswitch-test-plan.md

```diff
@@ -0,0 +1,876 @@
+# Test plan for Smartswitch
+
+- [Introduction](#introduction)
+- [Scope](#scope)
+- [Definitions and Abbreviations](#definitions-and-abbreviations)
+- [Test Cases](#test-cases)
+    - [1.1 Check platform inventory](#11-check-platform-inventory)
+    - [1.2 Check platform voltage](#12-check-platform-voltage)
+    - [1.3 Check platform temperature](#13-check-platform-temperature)
+    - [1.4 Check dpu console](#14-check-dpu-console)
+    - [1.5 Check midplane ip address between NPU and DPU](#15-check-midplane-ip-address-between-NPU-and-DPU)
+    - [1.6 Check DPU shutdown and power up individually](#16-check-DPU-shutdown-and-power-up-individually)
+    - [1.7 Check removal of pcie link between npu and dpu](#17-check-removal-of-pcie-link-between-npu-and-dpu)
+    - [1.8 Check the NTP date and timezone between DPU and NPU](#18-check-the-ntp-date-and-timezone-between-dpu-and-npu)
+    - [1.9 Check the State of DPUs](#19-check-the-state-of-dpus)
+    - [1.10 Check the Health of DPUs](#110-check-the-health-of-dpus)
+    - [1.11 Check reboot cause history](#111-check-reboot-cause-history)
+    - [1.12 Check the DPU state after OS reboot](#112-check-the-dpu-state-after-os-reboot)
+
+## Introduction
+
+The purpose is to test the functionality of Smartswitch.
+Smartswitch is connected to dpu sleds via pcie link having two dpus per sled.
```

Comment:
We don't have the sleds in our platform, could you please update to make it common for all vendors?

Reply:
Changed it.